resources/java_memory_assistant/bin/kill-upload-to-s3.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Encoding: utf-8
+# Cloud Foundry Java Buildpack
+# Copyright 2013-2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Kill upload of a file to S3
+# (c) Michele Mancioppi <michele.mancioppi@sap.com>
+
+set -eu
```

Comment:
There should be some consistency in the `#!` and `set` lines across all of these files.